scripts/tv-verify-all.sh

```
@@ -0,0 +1,243 @@
+#!/usr/bin/env bash
+# TradingView Indicators verification playbook runner
+set -uo pipefail
+cd "$(dirname "$0")/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)